src/components/Datepicker/Datepicker.const.tsx

```diff
@@ -0,0 +1,22 @@
+import { addYears, endOfYear } from 'date-fns/esm';
+import { nlBE } from 'date-fns/locale';
+import { ReactDatePickerProps } from 'react-datepicker';
+
+const base: Partial<ReactDatePickerProps> = {
```

Comment:
Ik denk dat we deze eerder vanuit de client moeten definiëren, dan moeten we hier ook al geen `date-fns` of locales toevoegen.
Idem voor `future` en `historic` dat gaat al meer over business logic waarvoor we de Datepicker gaan nodig hebben.

Reply:
Ze verplaatsen zorgt voor gedupliceerde configuratie (zoals stories momenteel) dus daar ben ik niet echt fan van, ik zal alvast de dependencies op die libraries er uit halen en de PR nog even laten staan.

Reply:
Hoezo gedupliceerd, dit zijn in principe default props toch? Dus je kan de `base` (hernoemen naar iets meer descriptive) idd gebruiken om in de Datepicker de defaults te zetten buiten dan de local (dit moet wel naar de client verhuizen). Die future en historic zijn dan voor de mocks waar de waardes ingesteld kunnen worden zoals gewenst.
```typescript
const Datepicker: FC<DatepickerProps> = ({
	calendarStartDay =  DEFAULT_DATEPICKER_PROPS.calendarStartDay,
	showYearDropdown =  DEFAULT_DATEPICKER_PROPS.showYearDropdown,
	showMonthDropdown =  DEFAULT_DATEPICKER_PROPS.showMonthDropdown,
	dropdownMode =  DEFAULT_DATEPICKER_PROPS.dropdownMode,
	...rest,
}) => {...}
```

Reply:
Dependencies verwijderd in #25 (https://github.com/viaacode/react-components/pull/25/commits/0a4b00d1462dc4819910d444927f04a9cc473de4).

Als we de rest van `src/components/Datepicker/Datepicker.const.tsx` verplaatsen naar de client dan zitten we met twee `Datepicker.const.tsx` files omdat `src/components/Datepicker/Datepicker.stories.tsx` ook gebruik maakt van diezelfde constants, daar zit de duplicatie die ik aanhaal.